src/legacy/legacy-default-xpring-client.ts

```diff
@@ -7,7 +7,8 @@ import { GetTransactionStatusRequest } from "../../generated/legacy/get_transact
 import { Payment } from "../../generated/legacy/payment_pb";
 import { SubmitSignedTransactionRequest } from "../../generated/legacy/submit_signed_transaction_request_pb";
 import { Transaction } from "../../generated/legacy/transaction_pb";
-import { TransactionStatus as RawTransactionStatus } from "../../generated/legacy/transaction_status_pb";
+import { TransactionStatus as RawTransactionStatusResponse } from "../../generated/legacy/transaction_status_pb";
```

Comment:
I don't see anywhere in the PR diff that `RawTransactionStatusResponse` is used. Do we need this import?